src/python/pants/goal/anonymous_telemetry_integration_test.py

```diff
@@ -7,7 +7,7 @@
 
 _no_explicit_setting_msg = "An explicit setting will get rid of this message"
 _no_repo_id_msg = 'set `repo_id = "<uuid>"` in the [anonymous-telemetry] section of pants.toml'
-_bad_repo_id_msg = "The repo_id must be between 30 and 60 characters long"
+_bad_repo_id_msg = "must be between 30 and 60 characters long"
 
 
 def test_warn_if_no_explicit_setting() -> None:
```

Comment:
To validate that this is fixed, dropping `use_pantsd` from any of these that are expecting to fail would be good. Since the message is in the constructor, it should be rendered deterministically before the run actually starts.

Reply:
Good idea, done.